### ISSUE TITLE
Documentation for 3D

### ISSUE THREAD

**Issue by tischi** [MouseLand/cellpose#252]
@carsen-stringer 
I was told that certain settings are ignored for the 3D case. 
But I could not find this information here: https://cellpose.readthedocs.io/en/latest/settings.html
Could this be added?

**Comment by carsen-stringer** [comment]
The flow_threshold is not used in 3D computation

**Comment by tischi** [comment]
@carsen-stringer thank you very much for the clarification, my suggestion was to _add_ the information that the flow_threshold is not used in 3D computation to the _documentation_. Do you think that could be helpful for the users?

Currently [this](https://cellpose.readthedocs.io/en/latest/settings.html) reads:

> Flow threshold (aka model fit threshold in GUI)

> Note there is nothing keeping the neural network from predicting horizontal and vertical flows that do not correspond to any real shapes at all. In practice, most predicted flows are consistent with real shapes, because the network was only trained on image flows that are consistent with real shapes, but sometimes when the network is uncertain it may output inconsistent flows. To check that the recovered shapes after the flow dynamics step are consistent with real masks, we recompute the flow gradients for these putative predicted masks, and compute the mean squared error between them and the flows predicted by the network.

> The flow_threshold parameter is the maximum allowed error of the flows for each mask. The default is flow_threshold=0.4. Increase this threshold if cellpose is not returning as many masks as you’d expect. Similarly, decrease this threshold if cellpose is returning too many ill-shaped masks.

Suggestion to add sentence: "Please note that the flow_threshold is not used in 3D computation, because..."
